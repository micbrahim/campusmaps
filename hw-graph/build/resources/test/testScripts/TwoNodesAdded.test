# tests adding two nodes to graph

# create empty graph
CreateGraph g1

# add two nodes
AddNode g1 n1
AddNode g1 n2

# print nodes in graph
ListNodes g1