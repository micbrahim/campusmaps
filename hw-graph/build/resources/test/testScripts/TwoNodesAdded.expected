# tests adding two nodes to graph

# create empty graph
created graph g1

# add two nodes
added node n1 to g1
added node n2 to g1

# print nodes in graph
g1 contains: n1 n2